### PR TITLE
Feature/title layer panel toggle

### PR DESCRIPTION
**Pull request by fungjj92**
This is my progress so far on #44. Putting it up to PR because the following tasks will just build on it and be more complicated so I'd like to get this in good shape first.

Should accomplish:
- [ ] Move Map Title into separate Map Layer, name should still be editable (p. 2)
- [ ] Move Side-panel down so it is below the top toolbar (using Map Layer card styling) (p. 2)
- [ ] Clicking Map Layer Collapse icon to collapse side panel. When side panel is collapsed, arrow icon should change to point in opposite direction (p. 4)

New bugs:
- [ ] Sliding open the side panel makes the toggle button disappear for a second
- [ ] Little 'share' button on the map slides around

Comments:
The toggle button styling is almost certainly not the final desired. I used a cartodb sprite.
I feel shaky moving into migrating metadata and options into the title tab.

In the process of fixing all the broken tests and writing a few new ones

### CODE REVIEW

**Comment by CloudNiner** [comment]
Is this rebased against current `bbg_production` branch? I pulled down to test and after updating the js I'm getting an error where the InsetMap view won't initialize. I have a bunch of diffs in the generated compressed cartodb.js files, so maybe the subproject commit on this branch is out of date?

File: lib/assets/javascripts/cartodb/table/actions_menu.js [hunk 1]
```diff
@@ -602,12 +726,51 @@
       this._manageLayers();
     },
 
-    _toggle: function(modName) {
-      // only hide if we click on active tab
-      if (this.model.get('open') && modName == this.getActivePane().panels.activeTab) {
-        this.hide(modName);
+    _showPanel: function() {
+      var panel_width = 450;
```
Comment (CloudNiner) [inline lib/assets/javascripts/cartodb/table/actions_menu.js]: Should probably make this a named constant in this view class.

File: lib/assets/javascripts/cdb [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit a2b8834622f2b4d3b849a7e810cff998d982844d
+Subproject commit 5bee7c8c6c6b2a62ec29c598f9c4331a9cb831fb
```
Comment (CloudNiner) [inline lib/assets/javascripts/cdb]: This shouldn't have updated if you didn't make any changes to cartodb.js, which it doesn't look like you did.

Comment (fungjj92) [inline lib/assets/javascripts/cdb]: oy vey

Comment (fungjj92) [inline lib/assets/javascripts/cdb]: I'm definitely rebased to the updated bbg_production branch. What do I have to do to pull in our custom cartodb.js, however?

Comment (CloudNiner) [inline lib/assets/javascripts/cdb]: You'll want to make sure that your git work tree is clean (no diffs if you `git status`), and then do a `git submodule sync && git submodule update`. We can take a look in the morning if that doesn't cut it.

**Comment by CloudNiner** [comment]
What's here LGTM, consider the named constant. 

Using the existing sprite is ok, bloomberg designers will replace with proper images. 

If you haven't solved the flickering button by now, maybe add a TODO in the code and we'll come back to it.

**Comment by fungjj92** [comment]
COOL doing it
Note: deleted no longer relevant tests, tried to add a few but they're broken to. TODO later, with more time.

**Pull request merged**